apps/api/src/info-request/dto/create-info-request.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { Expose } from 'class-transformer'
+import { IsString } from 'class-validator'
+
+export class CreateInfoRequestDto {
```

Comment:
This class actually copies the functionality of `CreateInquiryDto`

https://github.com/podkrepi-bg/api/blob/c4355b6f8221dc7a0e80ab235c55e142f3493174/apps/api/src/support/dto/create-inquiry.dto.ts#L8

Let's leave them co-exsist for now and will check later on for refactoring